tests/SAML2/XML/saml/AuthnContextTest.php

```diff
@@ -69,10 +74,7 @@ public function testMarshallingWithoutClassRef(): void
         );
 
         $document = DOMDocumentFactory::fromString('<root />');
-        /**
-         * @var \DOMElement $document->firstChild
-         * @var \DOMElement $authnContextElement
-         */
+        /** @var \DOMElement $document->firstChild */
         $authnContextElement = $authnContext->toXML($document->firstChild);
```

Comment:
This should use `$document->documentElement`, since `firstChild` is not guaranteed to be a `DOMElement`.

Reply:
Done